opslevel/resource_opslevel_service_repository.go

```diff
@@ -225,6 +225,10 @@ func (r *ServiceRepositoryResource) Read(ctx context.Context, req resource.ReadR
 			break
 		}
 	}
+	if serviceRepository == nil {
+		resp.Diagnostics.AddError("opslevel client error", fmt.Sprintf("expected ServiceRepository not found for service '%s'", currentStateModel.Service.ValueString()))
```

Comment:
So I think this is different then what happened before.  Before it would `d.SetID("")` which effectively means "it no longer exists" that would then signal to the SDK V1 that it needs to issue a "create" 

This will just throw and unrecoverable error.  No matter what they do.